harness/propertyHelper.js

```diff
@@ -1,5 +1,6 @@
 
 function isConfigurable(obj, name) {
+    var desc = Object.getOwnPropertyDescriptor(obj, name);
```

Comment:
The function is using `delete` instead of just `Object.getOwnPropertyDescriptor` to catch implementation errors where properties are reported as configurable through the `Object.getOwnPropertyDescriptor` API, but act like non-configurable when `delete` is used. 

(We should totally add comments to the propertyHelper.js functions to explain why the `Object.getOwnPropertyDescriptor` API isn't sufficient for the use cases in test262.)